ext/fetch/26_fetch.js

```diff
@@ -333,8 +335,18 @@
         response.body = null;
         core.close(resp.responseRid);
       } else {
+        const contentLengthHeader = ArrayPrototypeFind(
+          resp.headers,
+          (entry) => byteLowerCase(entry[0]) === "content-length",
+        );
```

Comment:
Faster way to do this that will have lesser impact on non-content-length header situations -

We already iterate over headers in the Rust op (`op_fetch_send`). Do this check in Rust and return `contentLength` value in the result.
 https://github.com/denoland/deno/blob/24f7f3fda984e7226b5858085a7ed1c53ab081e0/ext/fetch/lib.rs#L390-L392

Reply:
Don't know any rust tbh. I could give it a try.

Although we are already looping through headers on javascript side to match `location`:

https://github.com/denoland/deno/blob/65d2393c02a4ab637dd59149eb337631046346cc/ext/fetch/26_fetch.js#L371-L374

PS: different path, we're not looping twice.